assets/app/scripts/controllers/edit/buildConfig.js

```diff
@@ -107,6 +107,23 @@ angular.module('openshiftConsole')
       builderImageChangeTrigger: {},
       imageChangeTriggers: []
     };
+    $scope.runPolicyTypes = [
```

Comment:
we typically don't put something like this in the controllers, we use an ng-switch on the type in the view